frontend/app/forms/MemberForm.scala

```diff
@@ -119,7 +138,8 @@ object MemberForm {
       "marketingChoices" -> marketingChoicesMapping,
       "password" -> optional(nonEmptyText),
       "casId" -> optional(nonEmptyText),
-      "subscriberOffer" -> default(boolean, false)
+      "subscriberOffer" -> default(boolean, false),
+      "featureChoice" -> productFeature
```

Comment:
I believe that tampering with the form would not be that easy, as we [keep the member tier into account](https://github.com/guardian/membership-frontend/blob/books-or-events-choice/frontend/app/services/SubscriptionService.scala#L88) before assigning product features to   a Zuora subscription. Is just that we do not perform that check as part of the form validation but directly within the `SubscriptionService`